daemon/api.go

```diff
@@ -714,24 +687,50 @@ func modeFlags(devMode, jailMode bool) (snapstate.Flags, error) {
 
 }
 
+func snapUpdateMany(inst *snapInstruction, st *state.State) (msg string, updated []string, tasksets []*state.TaskSet, err error) {
+	// TODO: check inst flags and bail if any are given (-many
+	// doesn't take options)
+	updated, tts, err := snapstateUpdateMany(st, inst.Snaps, inst.userID)
```

Comment:
What's tts?  Text-To-Speech? :)

We've apparently never used that before, and it's not readable or hinting at anything else.

Can we please avoid it?  This same function uses tasksets for the same exact same variable when returning it. That's much better.